trellis/core/publisher.hpp

```diff
@@ -32,10 +32,13 @@ class PublisherClass {
  public:
   PublisherClass(const std::string& topic) : ecal_pub_(topic), ecal_pub_raw_(CreateRawPublisher(topic)) {}
   void Send(const MSG_T& msg) {
+    const auto now = trellis::core::time::Now();
+    Send(msg, now);
```

Comment:
Any risk that this will introduce any jitter in sort-of-deterministic repeaters?